app/common/state/tabState.js

```diff
@@ -499,7 +499,15 @@ const tabState = {
   },
 
   getOpenerTabId: (state, tabId) => {
-    return tabState.getTabPropertyByTabId(state, tabId, 'openerTabId', tabState.TAB_ID_NONE)
+    const openerTabId = tabState.getTabPropertyByTabId(state, tabId, 'openerTabId', tabState.TAB_ID_NONE)
+    if (openerTabId !== tabState.TAB_ID_NONE) {
+      // Validate that tabId exists
+      const index = getTabInternalIndexByTabId(state, openerTabId)
```

Comment:
I would probably use `getByTabId` here and it already handles TAB_ID_NONE by returning null. `getTabInternalIndex` is just a temporary hack until we can re-organize the tab state so best to limit its usage to the smallest set of methods possible